### PR TITLE
Based on 3GPP TS 29.060 version 7.6.0 Release 7

### PR DESCRIPTION
**Pull request by jbdamiano**
Section 7.7.49
the Apn restriction has a value from 0 to 3

7.7.49 APN Restriction 
The APN Restriction information element, when used in messages from the GGSN to the SGSN, contains an unsigned 
integer value indicating the level of restriction imposed on primary PDP Contexts created to the associated APN. When 
used in messages from the SGSN to GGSN, it contains an unsigned integer value indicating the highest level of 
ETSI  restriction type for all currently active PDP Contexts associated with the subscriber. In both cases, the meaning of the 
value contained within the IE is as defined in 3GPP TS 23.060 [4], sub-clause 15.4. 
The structure of the APN Restriction IE is as follows: 
 Bits 
 Octets    1      Type = 149 (Decimal) 
 Octets 2-3      Length 
 Octets   4       Restriction Type value

### CODE REVIEW

File: src/gtp_packet.erl [hunk 1]
```diff
@@ -1644,8 +1644,10 @@ decode_v1_element(<<M_flags_dual_address_bearer_flag:1,
     #common_flags{instance = Instance,
 		  flags = [ 'Dual Address Bearer Flag' || M_flags_dual_address_bearer_flag =/= 0 ] ++ [ 'Upgrade QoS Supported' || M_flags_upgrade_qos_supported =/= 0 ] ++ [ 'NRSN' || M_flags_nrsn =/= 0 ] ++ [ 'No QoS negotiation' || M_flags_no_qos_negotiation =/= 0 ] ++ [ 'MBMS Counting Information' || M_flags_mbms_counting_information =/= 0 ] ++ [ 'RAN Procedures Ready' || M_flags_ran_procedures_ready =/= 0 ] ++ [ 'MBMS Service Type' || M_flags_mbms_service_type =/= 0 ] ++ [ 'Prohibit Payload Compression' || M_flags_prohibit_payload_compression =/= 0 ]};
 
-decode_v1_element(<<>>, 149, Instance) ->
-    #apn_restriction{instance = Instance};
+decode_v1_element(<<M_restriction_type_value:8/integer,
+	_/binary>>, 149, Instance) ->
+	#apn_restriction{instance = Instance,
+		restriction_type_value = M_restriction_type_value};
```
Comment (RoadRunnr) [inline src/gtp_packet.erl:1650]: That part of the code is auto generated by priv/ie_gen_v1.erl. You have to adjust the declaration there and regenerate the code.

**Review by RoadRunnr — CHANGES_REQUESTED** [review]

File: src/gtp_packet.erl (hunk 1 above)
Comment (jbdamiano) [reply src/gtp_packet.erl]: I updated the file and regenerated the code

Comment (RoadRunnr) [reply src/gtp_packet.erl]: And please also adjust the IE test in test/property_test/gtplib_prop.erl

Comment (jbdamiano) [reply src/gtp_packet.erl]: Done

===> Running Common Test suites...
%%% gtp_SUITE: ...........
All 11 tests passed.

**Pull request merged**